lib/stdlib/src/math.erl

```diff
@@ -134,8 +134,31 @@ tan(_) ->
 tanh(_) ->
     erlang:nif_error(undef).
 
+-spec ceilf(X) -> float() when
+      X :: number().
+ceilf(_) ->
+    erlang:nif_error(undef).
+
+-spec floorf(X) -> float() when
+      X :: number().
+floorf(_) ->
+    erlang:nif_error(undef).
+
 %%% End of BIFs
 
+-spec ceil(X) -> integer() when
```

Comment:
I added a basic test at https://github.com/okeuday/erlbench/commit/5c7f8601940799acae6ced2f868724d625b0e581 and found that the time spent is consistently higher when doing 2 function calls, when compared to 1 function call with an if statement.  Since adding extra BIFs is not desirable, it makes sense to avoid adding the extra functions (especially when you consider the special modifications that are required to yield, and how that might matter with bignum usage).